qa/L0_large_payload/large_payload_test.py

```diff
@@ -59,72 +69,97 @@ def _test_helper(self,
                      model_name,
                      input_name='INPUT0',
                      output_name='OUTPUT0'):
+        # plan does not supoort large batch sizes.
+        # FIXME libtorch seems to have an issue with handling large batch sizes see DLIS-1770
+        if not (model_name.startswith('plan')
+                or model_name.startswith('libtorch')):
```

Comment:
I think we may just going to send 1.9 GB request to torch backend and let the test expect some kind of error are responded. And then we have the ticket to track the issue and update the test if it is fixed.

In this way we knows what exactly the server behaves in different cases, to avoid situation similar to HTTP where we assumes it works on large payload while it doesn't

Reply:
Fixed